README.md

```diff
@@ -0,0 +1,234 @@
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="doctors Icon.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Doctors Reservations Back-End</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Doctors Reservations Backend <a name="about-project"></a>
+
+
+**Doctors reservations Backend** is a back-end implementation of the front-end application that interacts with the back-end application to add doctors, logins, and reservations.
+
+## 🛠 Built With <a name="built-with"></a>
+- Rails
+- Ruby
+- Rspec
+- Api Documentation
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Implement the reservations,  using React and Ruby on rails**
+- **Demonstrate ability to apply best practices of communication for resolving teamwork challenges.**
+- **Add new doctors and reservations and using API REST to deploying in the Front-end Repository**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## Doctors Reservations Front-End
+
+[Front End App](https://github.com/JonahKayizzi/doctors-reservations-frontend.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+```sh
+ gem install rails
+ bundle install
+ rails db:create
+ rails db:migrate
+```
```

Comment:
- [ ] OPTIONAL: You have a `seeds.rb` in your project.  Kindly add to your instructions after `rails db:migrate` the command `rails db:seed` so your users can take advantage of it and have the opportunity to test the project with populated tables.